src/include/wiredtiger.in

```diff
@@ -49,12 +49,6 @@ extern "C" {
 #define	WT_ATTRIBUTE_LIBRARY_VISIBLE	__attribute__((visibility("default")))
 #endif
 
-#ifdef SWIG
```

Comment:
Was this deliberate? Did something change with SWIG meaning we don't need this any more?